dataactbroker/handlers/fileHandler.py

```diff
@@ -77,6 +78,7 @@ class FileHandler:
     FILE_TYPES = ["appropriations", "award_financial", "program_activity"]
     EXTERNAL_FILE_TYPES = ["award", "award_procurement", "executive_compensation", "sub_award"]
     VALIDATOR_RESPONSE_FILE = "validatorResponse"
+    UPLOAD_FOLDER = '/data-act/backend/tmp'
```

Comment:
We should be using env variables here, because this path is not used for every frontend dev env.